### PR TITLE
#120 - covizu iframe

### PR DESCRIPTION
**Pull request by samrichca**

### CODE REVIEW

File: components/NavBar.tsx [hunk 1]
```diff
@@ -82,6 +84,25 @@ const NavBar = (): ReactElement => {
...
+    border-radius: 4px;
+    padding: 2px 4px;
+    font-weight: normal;
+    margin-top: -12px;
+    margin-left: 4px;
+  `;
+
+  const [showCovizu, setShowCovizu] = useState(false);
+  useEffect(() => {
+    const nextShowCovizu = localStorage.getItem('SHOW_COVIZU') === 'true';
```
Comment (caravinci) [inline components/NavBar.tsx:102]: Is this a temporary test flag? will this eventually be implemented as an env-defined feature flag?

File: components/pages/visualization/index.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+import PageLayout from '../../PageLayout';
+
+const VisualizationPage = (): ReactElement => {
+  const theme: typeof defaultTheme = useTheme();
+  const { NEXT_PUBLIC_COVIZU_DATA_VERSION } = getConfig();
+
+  const [showCovizu, setShowCovizu] = useState(false);
+  useEffect(() => {
+    const nextShowCovizu = localStorage.getItem('SHOW_COVIZU') === 'true';
```
Comment (caravinci) [inline components/pages/visualization/index.tsx:34]: same question as before re. `feature` vs `test` flags

**Review by caravinci — APPROVED** [review]
🚢

File: components/NavBar.tsx (hunk 1 above)
Comment (samrichca) [reply components/NavBar.tsx:102]: temporary so we can turn it on in dev for testing

File: components/pages/visualization/index.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            css={css`
+              background: ${theme.colors.grey_2};
+              border: ${theme.colors.grey_3} 1px solid;
+              padding: 15px 20px;
+              margin: 15px;
+              border-radius: 10px;
+            `}
+          >
+            <a
+              href=""
```
[the remark this answers is not in the document]
Comment (samrichca) [reply components/pages/visualization/index.tsx:57]: yeah don't have content yet. good eye!

**Review by jaserud — APPROVED** [review]

**Pull request merged**